backend/handler_auth.go

```diff
@@ -116,7 +117,7 @@ func getFlow[F interface {
 	}
 	data.Messages = string(msgRaw)
 	if len(msg) == 0 {
-		data.Messages = "no messages right now"
+		data.Messages = "no error messages right now"
```

Comment:
Not sure if we want to display this text all the time? but we can keep it i guess :)